app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -162,21 +162,16 @@
   }
 }
 
-// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
   .gem-c-document-list__item {
     break-inside: avoid;
-  }
 
-  .gem-c-document-list__item-title {
-    .govuk-link {
-      color: $govuk-print-text-colour !important;
+    * {
+      color: $govuk-print-text-colour;
     }
+  }
 
-    .govuk-link::after {
-      display: block;
-      margin: 1mm auto;
-    }
+  .gem-c-document-list__item-context {
```

Comment:
Can you explain this change?

Reply:
For a document list with context, the context can wrap in a strange way depending on the length of the printed URL: 
<img width="" alt="image" src="https://github.com/user-attachments/assets/b24c9001-054f-4958-ae9f-87c00834d946">

Setting the context explicitly to `display: block` forces it to clear the space and start on a new line:
<img width="" alt="image" src="https://github.com/user-attachments/assets/4a241f6e-8ee8-4aec-bd08-fc1535ded370">

Reply:
Lines 169-171 force all the contents of this component to black. It's a new adjustment that I discovered was necessary because some elements of some variants render in a lighter grey otherwise, for example `.gem-c-document-list__attribute` is set to print in `#505a5f` unlesss we override it. I used the `*` selector to ensure that everything inside this component will be printed in black.